packages/py3_tomli_w.rb

```diff
@@ -0,0 +1,29 @@
+require 'package'
+
+class Py3_tomli_w < Package
+  description "Tomli-w is a lil' TOML writer."
+  homepage 'https://github.com/hukkin/tomli-w/'
+  @_ver = '1.0.0'
+  version @_ver
+  license 'MIT'
+  compatibility 'all'
+  source_url 'https://github.com/hukkin/tomli-w.git'
+  git_hashtag @_ver
+
+  binary_url({
+    i686: 'file:///usr/local/tmp/packages/py3_tomli_w-1.0.0-chromeos-i686.tar.zst',
+  x86_64: 'file:///usr/local/tmp/packages/py3_tomli_w-1.0.0-chromeos-x86_64.tar.zst'
```

Comment:
Local files?

Reply:
oops, that needs to be fixed, i'll do it when i boot up my PC tomorrow.